Makefile

```diff
@@ -145,10 +145,10 @@ ifeq ($(DEBUG),GDB)
 OPTIMISE_DEFAULT      := -Og
 
 LTO_FLAGS             := $(OPTIMISE_DEFAULT)
-DEBUG_FLAGS            = -ggdb3 -gdwarf-5 -DDEBUG
+DEBUG_FLAGS            = -ggdb2 -DDEBUG
```

Comment:
Why do we remove `gdwarf-5` here?

Reply:
Oops, I didn't push the latest branch as that change should not be in this PR, 1 sec...

Reply:
turns out I had made the change in my working copy but clearly got distracted by something else and hadn't ammended the commit or pushed the branch.  doh!  I've updated the branch now.